Add CHANGELOG and new README (#426)
bufbuild/vscode-buf: CHANGELOG.md

# Changelog

## Unreleased

> This release contains a rework of Buf for Visual Studio Code. It integrates the Buf Language

Reviewer: ```suggestion
This release reworks Buf for Visual Studio Code. It integrates the Buf Language
```

Server, available in [beta on the `buf` CLI][buf-lsp], introducing new [language server enabled features](README.md#features).

Thanks to [@christogav](https://github.com/christogav) for their contributions on integrating
the Buf Language Server with VS Code on this release.

## v0.7.2

- Fix issue with `buf` commands returning an unexpected error on exec.

## v0.7.1

- Fix "undefined" error when running `buf` on the extension.

## v0.7.0

- Output errors to "Buf" channel in the VS Code output console.

## v0.6.2

- Reintroduce relative binary path support.

## v0.6.1

- Revert relative binary path support.

## v0.6.0

- Support relative binary path.

## v0.5.3

- Add syntax highlighting for `.proto` files.

## v0.5.2

- Fix lint highlighting issue outside of the current file.

## v0.5.1

> - Fix an issue with setting buf format as the default formatter for proto3 files.

Reviewer: ```suggestion
- Fix an issue with setting `buf format` as the default formatter for proto3 files.
```

## v0.5.0

> - Add formatting through using buf format. Defaults to format on save.

Reviewer: ```suggestion
- Add formatting using `buf format`. Defaults to format on save.
```
